exe/shopify_transporter

```diff
@@ -35,10 +35,17 @@ module ShopifyTransporter
     method_option :object, type: :string, required: true,
       desc: 'The object type (customer, product, or order) to export.',
       enum: %w(customer product order)
+    method_option :first_id, type: :numeric, required: true,
+      desc: 'The starting id of the object type you are trying to export.'
```

Comment:
These descriptions are not very descriptive. I'm not sure how to improve them, but I definitely think they need attention.

Reply:
I'm not sure what's the best wording here. Might be worth getting into the range query args in the new wiki.

Reply:
Based on some input from Jenna, I'll change this to:

```
first_id

The first product_id, customer_id, or order number for the range of objects you are trying to export.


last_id

The last product_id, customer_id, or order number for the range of objects you are trying to export.


batch_size

The number of records that are retrieved from the source platform at one time. Use a smaller batch size if you are running into performance issues extracting your data.
```

Reply:
Great updates.